test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -514,6 +523,14 @@ public void TestRunCompleteHandlerShouldWriteToConsoleIfTestsAborted()
         }
 
         [TestMethod]
+        public void TestRunCompleteHandlerShouldWriteToConsoleIfTestsAbortedWithoutRunningAnyTest()
```

Comment:
Should we add tests for this change?
https://github.com/Microsoft/vstest/pull/1133/files#diff-f550c5f97286ac7d288d9c83547cff40R451
https://github.com/Microsoft/vstest/pull/1133/files#diff-f550c5f97286ac7d288d9c83547cff40R455

Reply:
We already have test for all the above mention scenario.
https://github.com/Microsoft/vstest/pull/1133/files#diff-1f79d9ff9499d17db1e9f39c5d479602R447
https://github.com/Microsoft/vstest/pull/1133/files#diff-1f79d9ff9499d17db1e9f39c5d479602R447
https://github.com/Microsoft/vstest/pull/1133/files#diff-1f79d9ff9499d17db1e9f39c5d479602R534